### ISSUE TITLE
NET-1595 Fix S1110 FN: Single variable parenthesized lambda

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#9080]
The rule should raise also for these parenthesis:

```csharp
AcceptLambda((x) => x+1); // Noncompliant
AcceptLambda(x => x+1);   // Compliant
```

To consider: Should it be a separate rule?

**Comment by pavel-mikula-sonarsource** [comment]
Internal ticket [NET-1595](https://sonarsource.atlassian.net/browse/NET-1595)

[NET-1595]: https://sonarsource.atlassian.net/browse/NET-1595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by pavel-mikula-sonarsource** [comment]
Moved
